examples/demo-react-native/android/settings.gradle

```diff
@@ -3,14 +3,12 @@ rootProject.name = 'DetoxRNExample'
 
 include ':app'
 
-println("RNInfo: rootDir=$rootDir")
```

Comment:
on purpose?

Reply:
Yes. It is a wrong log. WE don't user a root dir any more